app/brave_main_delegate.cc

```diff
@@ -145,6 +149,8 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
       switches::kExtensionContentVerificationEnforceStrict);
   command_line.AppendSwitchASCII(switches::kExtensionsInstallVerification,
       "enforce");
+  command_line.AppendSwitchASCII(switches::kOriginTrialPublicKey,
```

Comment:
as mentioned in DM, for local testing purposes we should only overwrite this if the switch does not have an existing value